### ISSUE TITLE
:eafnosupport error

### ISSUE THREAD

**Issue by svetob** [lexmag/statix#5]
I've been trying to test out statix, but I get an :eafnosupport error when trying to report metrics.

I have the a simple test module with default config:
```
defmodule Statixtest do
  use Statix
end
```

When I try to run it in `iex -S mix` I get the error:

```
iex(1)> Statixtest.connect()
:ok
iex(2)> Statixtest.increment "foo"
{:error, :eafnosupport}
```

I have a StatsD UDP port open on localhost:8125, same as default config.

I am running Erlang R19, Elixir 1.3.2, on Mac OS X.

**Comment by lexmag** [comment]
Thanks for the report! It has been fixed in master: 991eac3dce071747cac4ba5087b677b7260d0e99.
I'll put new version on Hex tomorrow.

**Comment by svetob** [comment]
Impressive turnaround time :) thanks!
